holidays/countries/india.py

```diff
@@ -40,6 +40,8 @@ class India(
         * Tamil Nadu:
             * [Tamil Monthly Calendar](https://web.archive.org/web/20231228103352/https://www.tamildailycalendar.com/tamil_monthly_calendar.php)
             * [Tamil Calendar](https://web.archive.org/web/20250429125140/https://www.prokerala.com/general/calendar/tamilcalendar.php)
+        * gu:
```

Comment:
```suggestion
        * Gujarat :
```
And move this to be before Tamil Nadu ones 👍